src/util/deploy/get-state-change-from-polling.ts

```diff
@@ -8,10 +8,10 @@ import getDeploymentByIdOrThrow from './get-deployment-by-id-or-throw';
 const POLLING_INTERVAL = 5000;
 
 async function* getStatusChangeFromPolling(
-  now,
-  contextName,
-  idOrHost,
-  initialState
+  now: any,
```

Comment:
I think it's possible to just `import Now from './src/util'` and use it as type here

Reply:
Good call. See 57290bab.